CHANGELOG

```diff
@@ -22,6 +22,10 @@ Version 2.0 (2019-07-03)
     if a domain with "daylight" was specified but no token was found, to help
     users transition to the new URLs.
 
+  * Added a custom "user-agent" to the v5 client's headers, along with
+    functionality to allow users (and specifically the CLI scripts) to add a
+    suffix to the user-agent
```

Comment:
This strikes me as a mostly internal mechanism.  Do we really want to mention it in the changelog?

Reply:
I don't know.  It's external-facing functionality, and I think of changelogs as things that do include internal changes (in a sort of "no surprises, we're not trying to hide anything from you" sort of way).  Feel free to get other opinions though.

Reply:
I guess I just didn't know what the standard practice was for reporting on internal behavior in an external package.  I'm willing to believe that it belongs in the changelog.